docs/pages/ds-display_L3_Low-Level_TestSpecification.md

```diff
@@ -0,0 +1,185 @@
+# Device Settings Display L3 Low Level Test Specification and Procedure Documentation
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Acronyms, Terms and Abbreviations](#acronyms-terms-and-abbreviations)
+- [References](#references)
+- [Level 3 Test cases High Level Overview](#level-3-test-cases-high-level-overview)
+- [Level 3 Python Test Cases High Level Overview](#level-3-python-test-cases-high-level-overview)
+
+## Overview
+
+This document describes the L3 Low Level Test Specification and Procedure Documentation for the Device Settings Display module.
+
+### Acronyms, Terms and Abbreviations
+
+- `HAL`  \- Hardware Abstraction Layer, may include some common components
+- `UT`   \- Unit Test(s)
+- `OEM`  \- Original Equipment Manufacture
+- `L3`   \- Level 3 Testing
+- `DS`   \- Device Settings
+- `EDID` \- Extended Display Identification Data
+- `DUT`  \- Device Under Test
+- `RAFT` \- Rapid Automation Framework for Testing
+- `Y`    \- yes supported
+- `NA`   \- Not Supported
+
+
+### References
+
+- `High Level Test Specification` - [dsDisplay High Level TestSpec](ds-display-high-Level_TestSpec.md)
```

Comment:
update the correct full link with latest version

Reply:
Done